memote/suite/tests/test_thermodynamics.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Perform tests using eQuilibrator API on an instance of ``cobra.Model``."""
+
+from __future__ import absolute_import, division
+
+from sys import version_info
+
+import pytest
+
+if version_info[:2] < (3, 5):
+    pytest.skip("Thermodynamic tests require at least Python version 3.5.",
+                allow_module_level=True)
+
+import memote.support.thermodynamics as thermo # noqa
+import memote.support.basic as basic # noqa
+from memote.utils import (annotate, get_ids, truncate) # noqa
+
+
+@annotate(title="Thermodynamic Reversibility of Purely Metabolic Reactions",
+          format_type="percent")
+def test_find_incorrect_thermodynamic_reversibility(read_only_model):
+    u"""
+    Expect reversibility of metabolic reactions to agree with thermodynamics.
```

Comment:
What about irreversible reactions that under the assumed conditions are pointing in the wrong direction? I think we should have a test for that, too.

Reply:
Frankly, I'm not entirely sure how to cover that with just the ln_gamma metric. I believe this would require some sort of combined check of dG and ln_gamma? As far as I understand ln_gamma only tells me if a reaction is, in theory, reversible but I'm not sure if it takes the specific direction into account, or just looks at the reactants and products kinda separately. 

In the interest of meeting my sprint goal I've created a new card for this and would like to defer any further discussion to when we plan that task.

Please see #530